### PR TITLE
Mmmmm, smash head with plank

### PR DESCRIPTION
**Pull request by Scrambledeggs00**
# Document the changes in your pull request

Adds in the flagellants chains to the game, accessible via the chaplain's armament beacon. It is a hooded suit item with negative armor that, when toggled, has a slowdown that is based on the wearer's health and wounds. The more damage and wounds the wearer has, the faster they go, with it getting to neutral speed between 79%-50% health and capping out at bellow 9% health and 3 wounds. To allow this speed to matter, it gives immunity to damage slowdown while toggled.  Along with this, it has bubble alerts to tell the user when they gain or lose speed, as well as messages and a particle effect which appear when they have reached max speed.

Speed wise, max speed is just a little faster than Ephedrine.

Armor wise: MELEE = -15, BULLET = -15, LASER = -15, ENERGY = -15, BOMB = -15, BIO = -15, RAD = 0, FIRE = 0, ACID = 0

The suit is also fire and acid proof so that it isn't easily destroyed.

The suit also takes 10 seconds to toggle the hood on, the purpose of this being to prevent people from just slapping it on whenever they want to.

# Why is this good for the game?

Gives the chaplain a new thing to gimmick with :]

# Testing

The video I have showing the speed test is too big, though I can post it on the discord if needed. Besides that, I have tested it and, as far as I can tell, all the parts do work as intended.

# Spriting

Mob Sprites:
FlagelantesChainsMob

Inhand and Object Sprites:
FlagelantesChainsInHandObject

# Wiki Documentation

Its name is "flagellant's chains"
Its armor values are MELEE = -15, BULLET = -15, LASER = -15, ENERGY = -15, BOMB = -15, BIO = -15, RAD = 0, FIRE = 0, ACID = 0
Its slowdown changes based on the wearer's health and wounds
It gives damage slowdown immunity
Its takes 10 seconds to toggle the hood.
Its effects, such as the speed and damage slowdown immunity, only take place when the hood is toggled

# Changelog

:cl:  
rscadd: Adds the flagellant's chains to chaplain's armament beacon
imageadd: Adds sprites for the flagellant's chains
/:cl:

### CODE REVIEW

**Review by cuackles — APPROVED** [review]

**Comment by kooarbiter** [comment]
i feel like the fastest speed should be 20 health or less, 9 health means a shitter with a welder can end your day before you even know they mean woe to your person

**Comment by Scrambledeggs00** [comment]
> i feel like the fastest speed should be 20 health or less, 9 health means a shitter with a welder can end your day before you even know they mean woe to your person

Mostly did that to lessen the chance of balance council wanting a change. If they're ok with it then I wouldn't mind changing that.

**Comment by JamieD1** [comment]
Awaiting Balance council comment

File: code/game/objects/items/holy_armours.dm [hunk 1]
```diff
@@ -233,3 +233,208 @@
...
+
+
+/obj/item/clothing/suit/hooded/flagelantes_chains/proc/handle_wound_remove(mob/living/carbon/human/H, datum/wound/W, obj/item/bodypart/L)
+
+	SIGNAL_HANDLER
+
+	if(suittoggled) //Make sure it only checks when the hood is up
+		change_slowdown(H, slowdown, -1) //Change speed when losing a wound
+
+/obj/item/clothing/suit/hooded/flagelantes_chains/proc/change_slowdown(mob/living/carbon/human/H, starting_slowdown, wound)
```
Comment (Moltijoe) [inline code/game/objects/items/holy_armours.dm]:
```suggestion
/obj/item/clothing/suit/hooded/flagelantes_chains/proc/change_slowdown(mob/living/carbon/human/H, starting_slowdown)
```

shouldn't need that variable anymore

**Review by Moltijoe — CHANGES_REQUESTED** [review]

**Comment by Moltijoe** [comment]
Balance Council:
>Make it a 3 second do_after instead

**Review by Moltijoe — APPROVED** [review]

**Comment by Scrambledeggs00** [comment]
> Balance Council:
> 
> > Make it a 3 second do_after instead

Done

**Pull request merged**